tests/integration/conftest.py

```diff
@@ -1,8 +0,0 @@
-# -*- coding: utf-8 -*
-# SPDX-License-Identifier: BSD-3-Clause
-
-
-def pytest_addoption(parser):
-    parser.addoption(
-        "--provider", action="store", default="nm", help="Network provider"
```

Comment:
I have checked the commit c29a2bf70813c035fca1be1fdd7a06833a17e47d, is not the hook used with `tests/integration/test_ethernet.py` ?

Reply:
> I have checked the commit [c29a2bf](https://github.com/linux-system-roles/network/commit/c29a2bf70813c035fca1be1fdd7a06833a17e47d), is not the hook used with `tests/integration/test_ethernet.py` ?

I see it in that commit - but where/how is it used?  doing `git grep conftest` in the `network` role local git clone shows nothing

Reply:
It seems to me that the `tests_integration_pytest.yml`  failure is related to the changes, I think that the initialization hooks are the common usage of `conftest.py`.

1. https://docs.pytest.org/en/7.1.x/example/simple.html#pass-different-values-to-a-test-function-depending-on-command-line-options
2. https://docs.pytest.org/en/6.2.x/reference.html#initialization-hooks

Reply:
ok - I see - conftest.py is used internally by pytest - so no explicit reference - I added it back